src/lib/utils/utils.ts

```diff
@@ -193,3 +193,15 @@ export function formatNDate(dateString: string) {
   }).format(newDate);
   return formattedDate;
 }
+
+export function timeOfDayGreeting() {
+  const today = new Date();
+  const hour = today.getHours();
+  if (hour < 12) {
+    return "Good morning";
+  } else if (hour >= 12) {
+    return "Good day";
```

Comment:
There is a logical error in the` if-else` conditions. Specifically, the `else if (hour >= 12)` condition will always be true for hours 12 and above, so the `else if (hour >= 17)` condition will never be reached.